### PR TITLE
Makes bags of holding gib the user on boh detonation

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Dax Dupont
balance: BoH detonations now gib the user.
/:cl:

It doesn't seem right the user of a boh explosion gets off scott free especially when they just climb the ladder and use a hand tele, mine a little or whatever.
At least with autocloners there needs to be a autocloner and some more effort than printing two bags.

### CODE REVIEW

**Comment by cacogen** [comment]
What does dusting do?

**Comment by DaxDupont** [comment]
It turns them to ash like being smacked into a SM.
Seems thematic!

**Comment by blobbernaut** [comment]
There's a pretty good chance you'll just get gibbed by the singulo thing in the middle anyway, isn't there? And it's not like this is being used as an actual viable method of station destruction atm

**Comment by DaxDupont** [comment]
I almost never see people gibbed instantly by the singularity.

**Comment by blobbernaut** [comment]
Still, there's a chance, which limits the usefulness. It's more dangerous and less destructive than a TTV bomb, while requiring higher barriers to use dangerously, while also being available much later and relying on mining getting you minerals.

There's just no good reason for this tbh.

**Comment by Shadowflame909** [comment]
I'll have to disagree with this. If you BOH bomb people, it's not a guaranteed death. They can survive it. So why should the user not have the same chance of surviving it?

**Comment by ExcessiveUseOfCobblestone** [comment]
the gibbing was from when the device terrorized the entire station, is it really necessary to keep it even though we toned down the destruction potential?

**Review by ShizCalev — DISMISSED** [review]

File: code/datums/components/storage/concrete/bag_of_holding.dm [hunk 1]
```diff
@@ -14,6 +14,7 @@
 		to_chat(user, "<span class='danger'>The Bluespace interfaces of the two devices catastrophically malfunction!</span>")
 		qdel(W)
 		playsound(loccheck,'sound/effects/supermatter.ogg', 200, 1)
+		user.dust()
```
Comment (ShizCalev) [inline code/datums/components/storage/concrete/bag_of_holding.dm]: i'd prefer `user.gib(TRUE, TRUE, TRUE)` myself tbh.

**Comment by ShizCalev** [comment]
actually, hm. 1 sec

**Review by ShizCalev — APPROVED** [review]

**Comment by Mickyan** [comment]
The user gets the chance to make preparations and grab a space suit and/or pickaxe to make escape a likely possibility, other people don't get that and are likely to be stuck or die

Does BoH bombing even add anything to the game other than being a reference? Which is not to say it should be removed but the latest rework made it an actual viable tactic for antags to instantly wipe an entire area with a single click and it always feels cheap and virtually unavoidable. 
At least when it was guaranteed to kill the user it wasn't the go-to method to kill a target in a desperate last-ditch effort.

**Comment by Shadowflame909** [comment]
@Mickyan You end up on lavaland. You dont need a space suit for lavaland, you just need a breathmask and an oxygen tank. Which we all get at roundstart.

Also, The BOH bomber is stranded with you? What are they gonna do, sprout angel wings and fly away? The singulo BOH bomb was better in every way as it was a round ender. Now all you need to do is get a couple of lattices and call it a day. There's no preparations you can take that would give you a better chance of surviving then anyone else you BOH bomb.

**Comment by Mickyan** [comment]
you can't dig through stone with a breathmask

**Comment by Shadowflame909** [comment]
There's a flaw in your argument. If the BOH bomber is somehow able to escape. Couldn't other people just follow him? Also Borgs can dig...

**Comment by Mickyan** [comment]
The bomber can escape without being followed because he made sure to grab a spacesuit beforehand instead and take the space route.

**Comment by YakumoChen** [comment]
one *specific* borg can dig, Shadowflame, that's not an argument. 

The rip in reality is also much larger then the actual Singularity it used to spawn and it used to be moderately easy to escape if you were farther then 3 tiles from the bomber. Now it makes like a 12x12 hole that eats entire shuttles all at once and strands / kills everyone

**Comment by Shadowflame909** [comment]
Every sentence you've just said has some kind of false statement. Two Borg's can mine, ala the standard Borg you see every round. The tear in reality has never ended up in the middle of any rocks, whatsoever. The fact that your trying to argue that the singulo does less damage then an easily repairable rift is astounding. The only shuttle it destroys is hi Daniel. Lastly, I doubt that at the very least not 1 person who falls in the rift has 1 way to escape. A mining tool, a hand teleporter. Maybe even the cloner?

**Comment by blobbernaut** [comment]
Needs Salt PR tag

**Comment by ExcessiveUseOfCobblestone** [comment]
Then balance it appropriately rather than just lmao the user?

That just seems like a copout because you don't like getting boh'd bombed.

**Comment by AlexTheSysop42** [comment]
> The tear in reality has never ended up in the middle of any rocks, whatsoever. 

False, one time it even spawned in the necropolis

>singulo
>less destructive than large but static rift
>singulo

what

**Comment by Dorsisdwarf** [comment]
Because if the singulo is popped on a shuttle there's a fairly good chance it will fall off the sides, people have a chance to run the fuck away and hide in the shuttle corners etc, is what i presume they meant

**Comment by AlexTheSysop42** [comment]
While it is possible to run from a singulo, it's far more destructive than the rift

**Comment by Shadowflame909** [comment]
Arguing is dumb as I've noticed that no matter how much you argue on a topic, everyone 90 percent of the time will stay on the same side of the argument beforehand. But they will just hate each other more once it ends.

I just hope the maintainers see how this PR is effectively making the "Less Devastating BOH reaction" as the previous head maintainer made it and merged it for, even more useless. If you really want to destroy sec, why not just get a syndibomb/toxins bomb THAT WILL GUARANTEE A KILL, AND won't KILL YOU. 

I mean, you could already do what the BOH reaction does on shuttles with an average bomb, except it will kill people instead of just transporting them.

**Comment by DaxDupont** [comment]
TTV requires effort.
BoH is clicking two buttons.

**Comment by Shadowflame909** [comment]
@DaxDupont  About as much effort as repeating the steps of a guide book.

**Comment by ExcessiveUseOfCobblestone** [comment]
If there's a problem with a wiki that keeps reoccurring in PRs I don't understand why that keeps being attributed as a (topic of PR) problem and not a wiki issue.

**Comment by Shadowflame909** [comment]
The only reason I'm talking about the wiki is that of the "apparent" effort it takes to do things in ss13. If you want deadly chemical grenades, follow the wiki. If you want a deadly bomb in a couple minutes, follow the wiki.

I'm just saying, the only shuttle this can really take out is "Hi Daniel" but literally anything in the game can take out that dumb shuttle. Why would you even want to use a BOH bomb if this got merged? When you can waste at most 1 more minute making a TTV or a fluorosulphuric acid chemical grenade.

**Comment by DaxDupont** [comment]
Making proper bombs is more complex than the guide in most cases, not only that it takes way more effort to 1. gain access to all the doors needed in toxins 2. Fuck with canisters 3. Properly mix the gasses 4. Fuck with tanks 
Have you ever even made a ttv?

Compared to bohs:
0. RnD did all your work for you 15 minutes in
1. Smash a window
2. Press print boh twice

**Comment by DaxDupont** [comment]
Also I've seen a boh take out everyone on the meta station shuttle, nobody made it to centcom on high pop.

**Comment by Shadowflame909** [comment]
@DaxDupont I've seen a bomb do that too, but bombs don't end up guaranteeing the bomber won't make it to Centcom. 

Also, It's not that difficult at all to make a TTV. Unless it's like you're first time doing it, but then literally everything would be difficult. Let alone breaking into RND or using the chem machine for a potassium and water grenade. 

Also, you can circumvent breaking in, in general by just going to the Head of Personnel for a job change. I hope I also mentioned that BoH's don't usually end up killing everyone in the blast radius. Did I? At the very most it needs a delay, not an instant kill for the BOH user.

**Comment by DaxDupont** [comment]
@Shadowflame909 That's my fucking point, TTVs are way harder to fucking make which is why they are more powerful than two clicks of making BoHs. 

But whatever, the maintainers can decide.

**Comment by ExcessiveUseOfCobblestone** [comment]
The wiki being used for anything more than general overview needed to do the job (not to do it well) is a wiki issue (that I don't disagree with. The wiki could use some better moderation).

In regards to

>Why would you even want to use a BOH bomb if this got merged? When you can waste at most 1 more minute making a TTV or a fluorosulphuric acid chemical grenade.

- TTV takes time to make (even if you use the wiki which i'm 100% sure is outdated), certainly not a short time after the atmos changes.
- Grenades are unreliable unless you're playing fetch for better assemblies and a pain if you don't have the supreme chemmaster. Portable chemmasters stink without T4, and even then they're not great.

BoH's involve printing 2 of the same item and clicking them together. The only effort is getting to the point to be able to make BoH's, and that really just boils down to a waiting game. That's literally the argument for toning them down from singulo spawning.

That said, I think the chasms are toned down enough. We removed the singulo for the exact reason of "well sometimes people can escape with autocloner", we don't need to play double-jeopardy here 
>because you don't like getting boh'd bombed

**Comment by Shadowflame909** [comment]
I'm just speaking frankly, why would you BOH bomb if there's no pay off. Just to destroy a few tiles temporarily in exchange for getting yourself killed permanently? Useless stuff. BoH's being unescapable is ridiculous to even suggest. I've never seen anyone actually get stuck in a tear of the rift and not getting out of it. Someone will always be there to lend a hand.

In fact, I'd like to argue it incites more co-operation than any other antagonist weapon. You can't just do things on your own and will need help to get out of your jam. I'd also cite that, it's easier to just break into tech storage and get a smoke machine. Then to wait for RnD to press all the buttons and get the mats they need.

Also, about the access to chemistry part, you don't really need an emag for anything as long as your human and a borg or an ai is around.

**Comment by optimumtact** [comment]
This should be obvious, if you want to BoH grief you can go to the effort of building an autocloner, if you want to destroy parts of the station easily, make bombs

**Comment by pantherpusher** [comment]
Boy that sure is a lot of debate about whether or not creating a black hole right in front of you should kill you or not

**Comment by Shadowflame909** [comment]
It doesn't make a black hole on the z level anymore. It just transports a small radius of people to a z level with a black hole that doesn't move. So you have to throw people into it if you want to do any damage. The arguing stems from the fact that it's weaker entirely. Everyone knows that. Should it have the same drawback as when it was stronger? Maintainers think so. Then that's that.

**Comment by blobbernaut** [comment]
if you upvoted this your opinion is wrong

**Pull request merged**